src/integrations/stytch.ts

```diff
@@ -115,7 +124,12 @@ export async function stytchLoginOnLoad(featheryClient: any) {
     return authFn()
       .then(() => stytchSubmitAuthInfo(featheryClient))
       .catch((e: any) => {
-        authState.showError();
+        const type = e.error_type;
+        let errorMsg;
+        if (type === 'bad_domain_for_stytch_sdk')
```

Comment:
could handleLoginOrCreateErrors be used here too? And what if we just call it handleStytchLoginErrors

Reply:
hm, yeah I guess it probably makes sense to use it here too even though some of the error types can't happen for oauth, since I'm already using that fact to share the same fn between sms and email

Reply:
oh, I could use the same fn - but oauth doesn't error until after the redirect, which is why I had to update the LoginError component & why this is different